cmd/tableinit/main.go

```diff
@@ -9,9 +9,10 @@ import (
 )
 
 var (
-	GcloudProject = os.Getenv("GCLOUD_PROJECT")
-	GcloudDataset = os.Getenv("GCLOUD_DATASET")
-	GcloudTable   = os.Getenv("GCLOUD_TABLE")
+	GcloudProject    = os.Getenv("GCLOUD_PROJECT")
+	GcloudDataset    = os.Getenv("GCLOUD_DATASET")
+	GcloudTable      = os.Getenv("GCLOUD_TABLE")
```

Comment:
It might be worth putting a comment here explaining what `GcloudTable` is just to help our future selves. I could see myself being confused in six months when I read this code again.

Reply:
yep! added some comments